bazel/io_opentelemetry_cpp.patch

```diff
@@ -0,0 +1,31 @@
+From b0825190258f28a7e72b62dfc00835cc277a059e Mon Sep 17 00:00:00 2001
```

Comment:
thanks @agrawroh  - can we upstream this patch?

Reply:
Sure, let me create a PR

Reply:
Opened a PR here: https://github.com/open-telemetry/opentelemetry-cpp/pull/3384